app/views/simulations/new.html.erb

```diff
@@ -52,10 +53,17 @@
               class: 'form-control simulation-quantity'
             %>
           </td>
+          <td>
+            <button type="button" class="btn btn-danger simulation-delete-row" onclick="deleteRow(event)">
```

Comment:
Quick question, whats the differences from defining `onclick` and using `$(...).on('click', ...);` like I did in [#L6](https://github.com/DerekStride/social-simulator/blob/simulations-view/app/assets/javascripts/simulations.js#L6)? If there's no real difference can we choose one convention in the mean time?